C/CasADi/build_tarballs.jl

```diff
@@ -22,6 +22,11 @@ cd build
 export CXXFLAGS="-fPIC -std=c++11"
 export CFLAGS="${CFLAGS} -fPIC"
 
+if [[ "$target" == *-mingw* ]]; then
+    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/windows.patch
+    export CXX=clang++
```

Comment:
No, you should do https://docs.binarybuilder.org/stable/build_tips/#Using-GCC-on-macOS-and-FreeBSD